num256/src/lib.rs

```diff
@@ -250,6 +248,29 @@ impl CheckedMul for Int256 {
   }
 }
 
+impl fmt::Display for Int256 {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", &self.to_str_radix(16))
+        //changed to base 16 for ethereum
```

Comment:
Is it possible to make the base16 write function a different one from fmt? Currently, fmt is used a lot for debugging, so it would be good to preserve base 10 for it.